<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-FLAOSQM" failures="1" tests="3" name="Default test(failed)" time="0.148" errors="0" timestamp="2024-01-17T18:24:10 IST">
  <testcase classname="testNGDiscussion.TestCase2" name="Login" time="0.019">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.fail(Assert.java:96)
at testNGDiscussion.TestCase2.Login(TestCase2.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="testNGDiscussion.TestCase2" name="Admin" time="0.0">
    <skipped/>
  </testcase> <!-- Admin -->
  <testcase classname="testNGDiscussion.TestCase2" name="Logout" time="0.0">
    <skipped/>
  </testcase> <!-- Logout -->
  <testcase classname="testNGDiscussion.TestCase2" name="Dashboard" time="0.0">
    <ignored/>
  </testcase> <!-- Dashboard -->
</testsuite> <!-- Default test(failed) -->
